katex.webpack.js

```diff
@@ -0,0 +1,11 @@
+/**
+ * This is the webpack entry point for KaTeX. As flow[1] and jest[2] doesn't support
+ * CSS modules natively, a separate entry point is used and it is not flowtyped.
+ *
+ * [1] https://gist.github.com/lambdahands/d19e0da96285b749f0ef
+ * [2] https://facebook.github.io/jest/docs/en/webpack.html
+ */
+import './static/katex.less';
+import katex from './katex.js';
```

Comment:
Importing `katex.less` is necessary so that we can build a .css file from it based on the `katex` entry point, right?

Reply:
@kevinbarabash `webpack` `style-loader` inlines imported stylesheets into the output JS files, and when JS files are loaded they are injected to the style tag. This is why css files are not linked in html files.

Using `extract-text-plugin`, they can be extracted to .css files. So in production build where `extract-text-plugin` is not enabled, it builds .css files.

Reply:
Cool.  Thanks for the explanation.